src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,9 @@
+@keyframes moving {
```

Comment:
Naming is confusing a bit. Better call this file `_keyframes` or just put it near the `animation`. You do have not so many keyframes